.github/workflows/visual-diff.yml

```diff
@@ -0,0 +1,27 @@
+name: visual diff
+on:
+  # This allows PRs opened from forks to execute the Visual Snapshots
+  # since secrets are not passed to workflows when triggered via forks
+  # Using workflow_run is less preferred since it executes even when jobs
+  # do not pass
+  workflow_run:
+    workflows:
+      - acceptance
+    types:
+      - completed
+
+jobs:
+  visual-diff:
+    # This is the opposite condition as acceptance.yml/visual-diff
+    if: github.head_repository.full_name != 'getsentry/sentry'
```

Comment:
This job will not execute when opened within the upstream repository.

Reply:
The only difference with the original `visual-diff` is this line.

Reply:
Good fix